scss/_base_icon-definitions.scss

```
@@ -358,6 +358,32 @@
+}
+
+@mixin vf-icon-linux-prompt($color) {
+  background-image: vf-icon-linux-url($color);
+}
+
+@mixin vf-icon-linux-prompt-themed {
+  @include vf-themed-icon($light-value: vf-icon-get-link-url($colors--light-theme--icon), $dark-value: vf-icon-get-link-url($colors--dark-theme--icon));
```

There is a copy/paste issue here, so link icon is used instead of linux prompt


```suggestion
  @include vf-themed-icon($light-value: vf-icon-linux-url($colors--light-theme--icon), $dark-value: vf-icon-linux-url($colors--dark-theme--icon));
```